cmd/openshift-install/graph.go

```diff
@@ -65,8 +65,7 @@ func runGraphCmd(cmd *cobra.Command, args []string) error {
 }
 
 func addEdge(g *gographviz.Graph, parent string, asset asset.Asset) {
-	elem := reflect.TypeOf(asset).Elem()
-	name := fmt.Sprintf(`"%s"`, elem.Name())
+	name := fmt.Sprintf(`"%s"`, reflect.TypeOf(asset).Elem().String())
```

Comment:
Minor nit, but you can get the same effect with the more-compact:

```go
fmt.Sprintf("%q", reflect.TypeOf(asset).Elem())
```

Reply:
https://github.com/openshift/installer/pull/739/commits/05627a851bb44c8f52bf6baa737cd8858382d877 -> https://github.com/openshift/installer/pull/739/commits/9184f32ff8b4c79b3e073714c6495f5621291f9f